packages/react-native-client/android/src/main/java/org/membraneframework/reactnative/VideoPreviewView.kt

```diff
@@ -1,25 +1,32 @@
 package org.membraneframework.reactnative
 
+import android.animation.ValueAnimator
 import android.content.Context
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import com.fishjamcloud.client.media.CameraCapturer
 import com.fishjamcloud.client.media.LocalVideoTrack
 import com.fishjamcloud.client.ui.VideoTextureViewRenderer
+import com.fishjamcloud.client.utils.getEnumerator
 import expo.modules.kotlin.AppContext
-import expo.modules.kotlin.views.ExpoView
+import kotlinx.coroutines.cancel
 import org.webrtc.RendererCommon
 
 class VideoPreviewView(
   context: Context,
   appContext: AppContext
-) : ExpoView(context, appContext) {
+) : MirrorableView(context, appContext) {
```

Comment:
Using inheritance for this case doesn't feel nice but I don't know how to explain it 😅 in kotlin you can inherit after a single class, what if we want to inherit after something else?  Maybe VideoPreviewView should have a reference to something like MirrorViewAnimator?